src/main/java/org/springframework/data/elasticsearch/core/MappingBuilder.java

```diff
@@ -333,6 +355,28 @@ private static void addFieldMappingParameters(XContentBuilder builder, Object an
 		}
 	}
 
+	/**
+	 * Apply mapping for dynamic templates.
+	 *
+	 * @throws IOException
+	 */
+	private static void addDynamicTemplatesMapping(XContentBuilder builder, String jsonString) throws IOException {
+		JsonNode jsonNode = new ObjectMapper().readTree(jsonString).get("dynamic_templates");
+		if (jsonNode != null && isNotBlank(jsonNode.toString()) && jsonNode.isArray()){
```

Comment:
IMO there is no need to check for `isNotBlank(jsonNode.toString())` because `jsonNode.isArray()` should be enough.

Reply:
Good one. Applied.